utils/outputwriter/testsutils.go

```diff
@@ -23,21 +24,31 @@ var (
 type OutputTestCase struct {
 	name               string
 	writer             OutputWriter
-	expectedOutputPath string
-	expectedOutput     string
+	expectedOutputPath []string
+	expectedOutput     []string
 }
 
 type TestBodyResponse struct {
 	Body string `json:"body"`
 }
 
-func GetExpectedTestOutput(t *testing.T, testCase OutputTestCase) string {
-	if testCase.expectedOutputPath != "" {
-		return GetOutputFromFile(t, testCase.expectedOutputPath)
+func GetExpectedTestCaseOutput(t *testing.T, testCase OutputTestCase) []string {
+	if len(testCase.expectedOutputPath) > 0 {
+		content := make([]string, len(testCase.expectedOutputPath))
+		for i, path := range testCase.expectedOutputPath {
+			content[i] = GetOutputFromFile(t, path)
+		}
+		return content
 	}
 	return testCase.expectedOutput
 }
 
+func GetExpectedTestOutput(t *testing.T, testCase OutputTestCase) string {
+	out := GetExpectedTestCaseOutput(t, testCase)
+	require.Len(t, out, 1)
```

Comment:
Someone told me that using 'require' is a bit problematic since it will exit immidiatly upon failure and might miss some errors that happened earlier and did not make the test fail immidiatly